src/auto_archiver/archivers/twitter_archiver.py

```diff
@@ -147,10 +147,16 @@ def download_yt_dlp(self, item: Metadata, url: str, tweet_id: str) -> Union[Meta
         tie = TwitterIE(downloader)
         tweet = tie._extract_status(tweet_id)
         result = Metadata()
+        try:
+            timestamp = datetime.strptime(tweet["created_at"], "%a %b %d %H:%M:%S %z %Y")
+        except Exception as ex:
+            logger.warning(f"Failed to get timestamp: {type(ex).__name__} occurred. args: {ex.args}")
+            return False
```

Comment:
assumption here that if the datetime parse fails the response was actually broken, is it so?

I see this is the previous scenario but with more granular logging.

Reply:
Yes, I haven't changed the functionality, just made clearer logging (I think I got an error on this at one point, which is why I changed it for clearer logging). Something for us to discuss going forwards. I'll flag this on the project so I can discuss with @erinhmclark - i.e.: logging and how to deal with potentially broken/incomplete/unexpected responses (is an 'unexpected' response better than nothing?)

Reply:
OK, so running on an invalid tweet gives the following response from youtubedlp

```
(Pdb) !tweet
{'user': {}, 'card': {}, 'quoted_status': {}, 'retweeted_status': {}}
```

We need some kind of better validation to make sure the tweet is valid. Right now it'll fail because the timestamp isn't parsed correctly (it's empty) but that's a bit brittle. Will add it to this PR